<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QWidget" name="Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>358</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>IDASec</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="master_layout">
     <item>
      <widget class="QLabel" name="idasec_title">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resources.qrc">:/icons/icons/idasec_small.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="binsec_layout">
       <item>
        <widget class="QLabel" name="binsec_label">
         <property name="text">
          <string>Binsec IP:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="binsec_ip_field">
         <property name="text">
          <string>127.0.0.1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Port:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="binsec_port_field">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>5570</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="binsec_connect_button">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="binsec_label_status">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="pinsec_layout">
       <item>
        <widget class="QLabel" name="pinsec_label">
         <property name="text">
          <string>Pinsec IP:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="pinsec_ip_field"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Port:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="pinsec_port_field">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pinsec_connect_button">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="pinsec_label_status">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="infos_title_label">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Infos:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="infos_label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="utils_label">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Utils</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="ir_label">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>IR Decode:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="dba_decode_field">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Instruction Hex eg:85db</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="dba_decode_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Decode</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="here_decode_button">
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/icons/icons/open-iconic-master/png/3x/target-3x.png</normaloff>:/icons/icons/open-iconic-master/png/3x/target-3x.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>12</width>
           <height>12</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="ir_textarea">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans Mono</family>
        </font>
       </property>
       <property name="cursorWidth">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>dba_decode_button</sender>
   <signal>clicked()</signal>
   <receiver>ir_textarea</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>453</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
